### PR TITLE
Dashboard polishing

### PR DESCRIPTION
**Pull request by rhKashif**
- Fixed duplicate titles for visualisations
- Fixed headline figure for number of reviews (based on review_id instead of review_text now)

### CODE REVIEW

File: dashboard/pages/community.py [hunk 1]
```diff
@@ -875,6 +906,7 @@ def wordcloud_rows(wordcloud_one: Chart, wordcloud_two: Chart) -> None:
     conn = get_db_connection(config)
 
     game_df = get_database(conn)
+    print(game_df["review_id"].nunique())
```
Comment (gawsalya) [inline dashboard/pages/community.py]: why do have a print statement?

File: dashboard/pages/community.py [hunk 2]
```diff
@@ -401,8 +432,8 @@ def format_columns(df_releases: DataFrame) -> DataFrame:
         lambda x: f"£{x:.2f}")
     df_releases['Release Date'] = df_releases['Release Date'].dt.strftime(
         '%d/%m/%Y')
-    df_releases['Community Sentiment'] = df_releases['Community Sentiment'].apply(
-        lambda x: round(x, 2))
+    # df_releases['Community Sentiment'] = df_releases['Community Sentiment'].apply(
```
Comment (DMcCallion) [inline dashboard/pages/community.py]: Either get rid of this or uncomment it

File: dashboard/pages/community.py [hunk 3]
```diff
@@ -366,6 +381,18 @@ def aggregate_data(df_releases: DataFrame) -> DataFrame:
...
+    Normalize sentiment values to be strings and reduce to 3 significant figures
+
+    Args:
+        row (DataFrame): A DataFrame containing new release data
+    Returns:
+        DataFrame: A DataFrame containing new release data with aggregated data for each release
+    """
+    return str(sentiment)[:3]
+
+
```
Comment (angelikavilde) [inline dashboard/pages/community.py]: return type is wrong and so is the docstring

**Review by gawsalya — APPROVED** [review]
Just one change please - thanks 
The commented out code as well!

**Review by DMcCallion — APPROVED** [review]
Looks good, just minor quality improvements such as commented out code

**Pull request merged**